switchmap/poller/snmp/async_snmp_manager.py

```
@@ -0,0 +1,949 @@
+        try:
+            # Test if we can poll the SNMP sysObjectID
+            # if true, then the device is contactable
+            result = await self.sysobjectid(check_reachability=True)
+            if bool(result) is True:
+                contactable = True
+        
+        except Exception:
+            # Not Contactable
+            contactable = False
+        
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)